### PR TITLE
 add localization strings for module examples.js

### PR DESCRIPTION
**Pull request by ThobyV**
This P.R aims to apply proper changes requested to resolve issue https://github.com/w3c/respec/issues/2084 for the module `examples.js`. Thanks to all who review and @marcoscaceres for assistance.

### CODE REVIEW

**Comment by ThobyV** [comment]
Anyone knows how I can stop making old  commits follow me  everywhere. 😅

File: tests/spec/core/examples-spec.js [hunk 1]
```diff
@@ -148,3 +148,19 @@ describe("Core — Examples", () => {
     expect(mybutton.onclick).toBeDefined();
   });
 });
+
+it("makes sure examples section have expected localization text", async () => {
```
Comment (marcoscaceres) [inline tests/spec/core/examples-spec.js]:
```suggestion
it("localizes examples", async () => {
```

**Comment by ThobyV** [comment]
Oh.. Ok! I would love that. Would send a commit as per your suggestions too. Thanks  for the quick review.

**Comment by marcoscaceres** [comment]
@ThobyV do:

Do this once:
```
git config --global branch.autosetuprebase always
```

Then: 

```
git remote add upstream git@github.com:w3c/respec.git
git checkout develop 
git pull upstream develop
git push origin develop
```

That will get you tracking the this repository. Just do the above every time you want to start a new feature.

**Comment by marcoscaceres** [comment]
Code is live in the current version. Thanks again for your help with these @ThobyV!

**Comment by ThobyV** [comment]
Thanks for the review and useful git commands @marcoscaseres I'm grateful. I will use these from now onward. 😃

**Pull request merged**